parse-zoneinfo/README.md

```diff
@@ -1,42 +1,35 @@
 # parse-zoneinfo
 
-Parse-zoneinfo is a fork of [`zoneinfo_parse`][zoneinfo_parse], with adjustments such that it no longer depends on the `Datetime` crate. It is used by [`chrono-tz`][chrono_tz].
+Rust library for reading the text files comprising the [zoneinfo database], which records time zone changes and offsets across the world from multiple sources.
```

Comment:
Can you turn this into smaller commits that do one thing? I don't know what I'm reviewing for here.

Reply:
Split into three commits.

Reply:
Sorry, but this first commit still looks like a grab bag of unrelated improvements to me. Want to split it some more?

Reply:
Done :disappointed:.